Prac_05/wimbledon.py

```diff
@@ -20,7 +20,7 @@ def get_records(filename):
 
 
 def process_data(records):
-    """Processes data into dictionary and set"""
+    """Processes data into dictionary and set."""
     countries = set()
     champion_to_times_won = {}
```

Comment:
I think this name is okay but I was a bit confused at first, maybe changing "times won" to a single word will improve clarification, like "champion_to_wins" or "champion_to_count"

Reply:
Also, all docstrings as well as commit messages should be written in the imperative mood, Lindsey has a whole lecture video on it which I found to be quite helpful :)